AirSkyBoat/AirSkyBoat
# [core] Line of sight fixes

**_I affirm:_**


## Please enter a player-facing description
- Fixes an issue where a player would run behind an object during a cure and the cure would cancel (Frank)
- Fixes an issue where a ranged attack would be **cancelled** if the mob walked behind an object you cannot see through. (Frank)


## What does this pull request do? (Please be technical)
These two function calls should be set to True. This allows this to work as intended.

Right now if a player is getting healed and walks behind a wall it will cancel. This will allow the cure to finish casting if that happens. The cure will STILL cancel if the player walks out of range.

Both tested on retail. This is also was LSB has upstream. 

Video of ranged attack missing behind a wall:
https://www.youtube.com/watch?v=7iBdomixR9U


## Steps to test these changes
Have two characters
Player 1 -> Cast Cure on Player 2
Player 2 -> Walks behind wall
See the cure still finish casting

Ranged attack a mob that is walking way from you
When it goes beyond 25 see that the attack will now finish the animation instead of cancelling



## Special Deployment Considerations
